catalogue/webapp/services/prismic/fetch/index.ts

```diff
@@ -1,23 +1,11 @@
 import * as prismic from '@prismicio/client';
 import { GraphQLClient } from 'graphql-request';
 import { PrismicApiError } from '../types';
-import { unCamelCase } from '@weco/common/utils/grammar';
-import { ArticleFormatIds } from '@weco/common/data/content-format-ids';
 import { Query } from '@weco/catalogue/types/search';
 import { getPrismicSortValue } from '@weco/common/utils/search';
 import { storiesQuery } from './articles';
 import { createClient as createPrismicClient } from '@weco/common/services/prismic/fetch';
 
-export const articleIdToLabel = (id: string): string => {
```

Comment:
Talking it out with Gareth, this wasn't necessary as we're getting the format returned at the root of the response object.